pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -0,0 +1,75 @@
+package loggingcredentials
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// LoggingCredentialsSecretMeta returns metadata for the logging-operator credentials secret.
+func LoggingCredentialsSecretMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Name:      "logging-credentials",
+		Namespace: "monitoring",
+	}
+}
```

Comment:
I think this could just be pair of constants for the namespace and name at the package level. Is there any advantage of having a function that always returns the same?

Reply:
So we could use this from other package later on.

Reply:
Can't you do that using constants?